util/Fs.cpp

```diff
@@ -252,7 +252,14 @@ ResourcePressure Fs::readRespressure(const std::string& path) {
     // aggr 316016073
     // some 0.00 0.03 0.05
     // full 0.00 0.03 0.05
-    OCHECK(lines.size() == 3);
+    if (lines.size() != 3) {
+      std::string exception_error;
+      for(unsigned int i=0;i<lines.size();i++)
```

Comment:
I don't think the loop is necessary. It was fine the way it was before. The change I suggested was exactly the change I wanted you to make. No more, no less.